<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInfoDoc</class>
 <widget class="QDialog" name="DialogInfoDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="nombre_doctor">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Especialidad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="doc_espec">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>especialidad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultorio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_consult">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>numero</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Horario de atención:</string>
   </property>
  </widget>
  <widget class="QLabel" name="turnos">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>tal ves previa cita</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubicación</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Datos de contacto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>numero</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_Tel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>numer delwey</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="correo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>correo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Formación profesional:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cedula:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cedula">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cedula del wey</string>
   </property>
  </widget>
  <widget class="QLabel" name="foto_doc">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>Foto del doctor</string>
   </property>
  </widget>
  <widget class="QLabel" name="hospital">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hospital</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
